qa/tasks/cephfs/test_cephfs_shell.py

```diff
@@ -335,6 +335,48 @@ def test_rmdir_p_dir_with_file(self):
         self.mount_a.stat(self.dir_name)
 
 
+class TestRm(TestCephFSShell):
+    dir_name = "test_rm"
+
+    def dir_does_not_exists(self):
+        """
+        Tests that directory does not exists
+        """
+        try:
+            self.mount_a.stat(self.dir_name)
+        except CommandFailedError as e:
+            if e.exitstatus == 2:
+                return 0
```

Comment:
```suggestion
            self.assertEqual(e.exitstatus, 2)
```

In case of failures, this makes it small bit easier to understand that we expect exit status to be 2.